File: enc.php
<?php
function decryptData($encrypted_data, $key) {
    // Base64 decode the encrypted data
    $data = base64_decode($encrypted_data);
    
    // Extract the IV (first 16 bytes) and the encrypted message
    $iv = substr($data, 0, 16);
    $encrypted_bytes = substr($data, 16);
    
    // Decrypt using AES-256-CBC with the given key and IV
    $decrypted = openssl_decrypt($encrypted_bytes, 'aes-256-cbc', $key, OPENSSL_RAW_DATA, $iv);
    
    // Remove padding if necessary
    $padding_length = ord(substr($decrypted, -1));
    return substr($decrypted, 0, -$padding_length);
}

// Example usage
$encrypted_data = "JS2W9X0eHUKi1iQycBRnIMdokpO1f5AZUPSXHJU8edo=";  // Replace with your encrypted data
$key = "your_secret_key_32_bytes";                    // Must be exactly 32 bytes for AES-256
$decrypted_text = decryptData($encrypted_data, $key);
echo $decrypted_text;
?>

File: enc.py
from Crypto.Cipher import AES
from Crypto.Random import get_random_bytes
from base64 import b64encode

def encrypt_data(plain_text, key):
    iv = get_random_bytes(16)  # 16-byte IV for AES
    cipher = AES.new(key, AES.MODE_CBC, iv)
    
    # Padding
    padding_length = 16 - len(plain_text) % 16
    padded_text = plain_text + chr(padding_length) * padding_length
    
    encrypted_bytes = cipher.encrypt(padded_text.encode('utf-8'))
    
    # Return base64 encoded IV + encrypted data
    return b64encode(iv + encrypted_bytes).decode('utf-8')

# Example usage
key = b'your_secret_key_32_bytes'  # 32 bytes key
plain_text = "Hello World"
encrypted_text = encrypt_data(plain_text, key)
print(f"Encrypted Text: {encrypted_text}")
